threedi_modelchecker/checks/timeseries.py

```diff
@@ -28,6 +36,93 @@ def description(self):
         return f"{self.column_name} contains an empty timeseries; remove the {self.table.name} instance or provide valid timeseries."
 
 
+class TimeSeriesEqualTimestepsCheck(BaseCheck):
+    """
+    Check that the timesteps in all timeseries in a column are equal.
+
+    This checks the timesteps for all timeseries in a column against the timesteps in the first
+    timeseries in that column. Consequently, if the first timeseries is wrong, all the other timeseries
+    in that column will raise a warning.
+    This check does not compare timeseries between different columns; for that, FirstTimeSeriesEqualTimestepsCheck
+    is used.
+    """
+
+    def get_invalid(self, session):
+        invalid_timeseries = []
+
+        first_timeseries = None
+
+        for row in self.to_check(session).all():
+            timeseries = row.timeseries
+
+            if not timeseries:
+                continue
+
+            if not first_timeseries:
+                first_timeseries = timeseries
+                continue  # don't compare first timeseries with itself
+
+            if not compare_timesteps(
```

Comment:
I am a bit worried that you get Exceptions here because the timeseries do not adhere to the correct format (see other checks).

I think a `try ... except Exception: pass` would be an OK solution here.